configuration/drugs/drugs.csv

```diff
@@ -12,4 +12,6 @@ a342020e-9e10-42d4-87de-d2904cccd507,0,ABC+3TC+EFV,ABC+3TC+EFV,,
 6f5cc94f-f705-4c1e-9a2e-6201cd8a9495,0,TDF+3TC+ATZ/r,TDF+3TC+ATZ/r,,
 9ba1ea9d-ad51-4e08-bb4c-3ca94d6826ad,0,TDF+3TC+LPVr,TDF+3TC+LPVr,,
 e49c8fd8-920b-41ff-93ae-10c5e93d90f3,0,AZT+DRV+RTV,AZT+DRV+RTV,,
-5a3bc174-0164-4ae6-8c4d-ba1311fd0a33,0,ABC+DRV+RTV,ABC+DRV+RTV,,
\ No newline at end of file
+5a3bc174-0164-4ae6-8c4d-ba1311fd0a33,0,ABC+DRV+RTV,ABC+DRV+RTV,,
+1426379c-9274-11e9-bc42-526af7764f64,0,NVP,NVP,,
+14263aa8-9274-11e9-bc42-526af7764f64,0,NVP child,NVP child,,
```

Comment:
@prashantsable1991 this should be `NVP (E)` according to the requirement 
![image](https://user-images.githubusercontent.com/22587098/59922787-d33fe880-9431-11e9-88c3-b878a830d508.png)